reoptjl/models.py

```diff
@@ -527,32 +549,46 @@ class SiteOutputs(BaseModel, models.Model):
         null=True, blank=True,
         help_text="Total tons of PM2.5 emissions associated with the site's onsite fuel burn over the analysis period."
     )
-    annual_renewable_electricity_kwh_bau = models.FloatField(
+    annual_onsite_renewable_electricity_kwh_bau = models.FloatField(
         null=True, blank=True,
         help_text=(
                   "Electricity consumption (incl. electric heating/cooling loads) that is derived from on-site renewable resource generation in the BAU case."
                   "Calculated as total RE electric generation in the BAU case, minus storage losses and curtailment, with the user selecting whether exported renewable generation is included). "
                   )
     )
-    renewable_electricity_fraction_bau = models.FloatField(
+    onsite_renewable_electricity_fraction_of_elec_load_bau = models.FloatField(
         null=True, blank=True,
         help_text=(
                   "Electricity consumption (incl. electric heating/cooling loads) that is derived from on-site renewable resource generation in the BAU case."
                   "Calculated as total annual RE electric generation in the BAU case, minus storage losses and curtailment, with the user selecting whether exported renewable generation is included, "
                   "divided by total annual electric consumption."
                   )
     )
-    total_renewable_energy_fraction_bau = models.FloatField(
+    onsite_renewable_energy_fraction_of_total_load_bau = models.FloatField(
         null=True, blank=True,
         help_text=(
                   "Portion of annual total energy consumption that is derived from on-site renewable resource generation in the BAU case."
-                  "The numerator is calculated as total annual RE electricity consumption (calculation described for annual_renewable_electricity_kwh_bau output),"
+                  "The numerator is calculated as total annual RE electricity consumption (calculation described for annual_onsite_renewable_electricity_kwh_bau output),"
                   "plus total annual thermal energy content of steam/hot water generated from renewable fuels (non-electrified heat loads)."
                   "The thermal energy content is calculated as total energy content of steam/hot water generation from renewable fuels,"
                   "minus waste heat generated by renewable fuels, minus any applicable hot water thermal energy storage efficiency losses."
                   "The denominator is calculated as total annual electricity consumption plus total annual thermal steam/hot water load."
                   )
     )
+    onsite_and_grid_renewable_electricity_fraction_of_elec_load_bau = models.FloatField(
+        null=True, blank=True,
+        help_text=(
+                  "Calculation is the same as onsite_renewable_electricity_fraction_of_elec_load, but additionally includes the renewable energy"
```

Comment:
maybe these help texts should reference the bau versions instead, i.e. onsite_renewable_electricity_fraction_of_elec_load_bau here and onsite_renewable_energy_fraction_of_total_load_bau below

Reply:
done in [ecd9e7d](https://github.com/NREL/REopt_API/pull/628/commits/ecd9e7d5467a9fc5d50f1cd050b7d953887de608)